libs/blockscout-client/crate/src/apis/blocks_api.rs

```diff
@@ -0,0 +1,209 @@
+/*
+ * BlockScout API
+ *
+ * API for BlockScout web app
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: you@your-company.com
```

Comment:
probably it should be set to some blockscout email (or removed)? nit

Reply:
I didnt file a way to indicate email in config for [openapi-generator](https://github.com/OpenAPITools/openapi-generator)